indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -678,14 +685,15 @@ public Void call() throws Exception
   @VisibleForTesting
   static void sortByInsertionTime(List<RemoteTaskRunnerWorkItem> tasks)
   {
-    Collections.sort(tasks, new Comparator<RemoteTaskRunnerWorkItem>()
-                     {
-                       @Override
-                       public int compare(RemoteTaskRunnerWorkItem o1, RemoteTaskRunnerWorkItem o2)
-                       {
-                         return o1.getQueueInsertionTime().compareTo(o2.getQueueInsertionTime());
-                       }
-                     }
+    Collections.sort(
+        tasks, new Comparator<RemoteTaskRunnerWorkItem>()
```

Comment:
Hm, I didn't modify this, just ran the code style check against the file, so I'm leaning towards leaving it as is to minimize changes.